stock_picking_show_linked/views/stock_picking.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data>
+        <record id="stock_picking_show_linked_form_inherit" model="ir.ui.view">
+            <field name="name">stock_picking_show_linked.form</field>
+            <field name="model">stock.picking</field>
+            <field name="inherit_id" ref="stock.view_picking_form" />
+            <field name="arch" type="xml">
+                <div name="button_box" position="inside">
+                    <field name="dest_picking_count" invisible="1" />
+                    <field name="origin_picking_count" invisible="1" />
+                    <button
+                        class="oe_stat_button"
+                        type="object"
+                        name="action_stock_picking_destination"
+                        icon="fa-folder-open"
```

Comment:
Thanks for the module, LGTM! Just one comment, why is the icon a folder? Wouldn't it be better to use the truck icon, like in the "Transfers" button? Also, as @LoisRForgeFlow suggested at #1443 maybe is better the name "Transfer" instead of "picking" to be more aligned with the typical UI nomenclature for stock.picking.

Reply:
@albariera done I changed the icon and strings

Reply:
Thanks!!